src/Lumi/Styles/Banner.purs

```diff
@@ -0,0 +1,31 @@
+module Lumi.Styles.Banner where
+
+import Prelude
+
+import Data.Foldable (fold)
+import Lumi.Components (PropsModifier)
+import Lumi.Styles (color, css, int, str, style)
+import Lumi.Styles.Border (_round) as S
+import Lumi.Styles.Box (FlexAlign(..), _align, _row, box) as S
+import Lumi.Styles.Responsive (desktopQuery)
+import Lumi.Styles.Theme (LumiTheme(..))
+
+banner :: forall props. PropsModifier props
+banner =
+  S.box
+  <<< S._row
+  <<< S._align S.Center
```

Comment:
I was testing this locally and felt that it was a bit weird to have the dismiss button aligned to the center on mobile. I thought that having `S._align S.Baseline <<< onDesktop (S._align S.Center)` was easier on the eye, what do you think?